src/utils/GitUtils.js

```diff
@@ -0,0 +1,30 @@
+const axios = require('axios')
+const fs = require('fs')
+
+module.exports = class GitUtils {
+  static async getHashOrBranch () {
+    const rev = fs.readFileSync('.git/HEAD').toString().trim()
```

Comment:
What happens if `.git/HEAD` doesn't exist?

Reply:
The only reason that I see that could happen is if there was no local repository, and on that case the command just becomes obsolete

Reply:
But anyway, I will take that into account

Reply:
https://github.com/SwitchbladeBot/switchblade/pull/1173/commits/578af3c5aea34bd53728c394f81a8e9dd9b8a443